docs/topics/ref_health-check-resources.adoc

```diff
@@ -1,16 +1,16 @@
 [id='health-check-resources_{context}']
 = Health check resources
 
-More background and related information on health checking can be found here:
+More background and related information on health checks can be found here:
 
 * link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-application-health[Application Health in OpenShift]
 
 * link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/[Kubernetes Liveness and Readiness Probes]
 
-ifndef::built-for-spring-boot[* link:{link-example-health-check-spring-boot}[{name-example-health-check} for {SpringBoot}]]
+ifndef::built-for-spring-boot,built-for-getting-started[* link:{link-example-health-check-spring-boot}[{name-example-health-check} for {SpringBoot}]]
```

Comment:
Why not show the example app links for individual runtimes in the Getting Started Guide?

Reply:
I think the example applications in the runtime guides are different than the capabilities provided by the launcher. While they are based on the same code, they don't have the same features.

Reply:
All right. Please verify with Engineering.